includes/polling/os/f5.inc.php

```diff
@@ -1,2 +1,4 @@
 <?php
-$version = trim(snmp_get($device, '.1.3.6.1.4.1.3375.2.1.4.2.0', '-OQv', '', ''), '"');
+$version = trim(snmp_get($device, '.1.3.6.1.4.1.3375.2.1.4.2.0', '-OQv'), '"');
+$hardware = snmp_get($device, '.1.3.6.1.4.1.3375.2.1.3.5.2.0', '-OQv');
```

Comment:
You will need to trim quotes from these as well.

The other way around that is to use the MIB aka `F5-BIGIP-SYSTEM-MIB::sysGeneralChassisSerialNum.0`